src/app.ts

```diff
@@ -1304,6 +1304,12 @@ function registerIoSignAPIRoutes(
     profileService
   );
 
+  app.get(
+    `${basePath}/metadata`,
+    bearerSessionTokenAuth,
+    constantExpressHandler(ioSignController.getMetadata())
```

Comment:
If we use a constant controller we should return the response without using a controller.
```suggestion
    constantExpressHandler(ResponseSuccessJson({
      serviceId: IO_SIGN_SERVICE_ID as NonEmptyString
    });)
```
Otherwise use the `toExpressHandler` middleware to apply correctly the controller class context to the method execution (the method should return a Promise).

Reply:
ok i removed the use of the controller!